### DISCUSSION TITLE
storybook-addon-baseline-grid: Decide where the addon should be available

### DISCUSSION THREAD

**Discussion by bartaz** [canonical/pragma#95] [Ideas]
Currently `storybook-addon-baseline-grid` addon button is available on individual Stories.
It doesn't work well in Docs pages (displays overlay on top of whole page instead of embedded stories), and baseline can't be toggled in full screen mode.

We should review the options for where the addon tool button can be available and implement improvements to how it works.

Related discussion: https://github.com/canonical/ds25/pull/86#discussion_r1893163078

**Comment by syncronize-issues-to-jira** [comment]
Thank you for reporting your feedback to us!

The internal ticket has been created: https://warthogs.atlassian.net/browse/WD-17962.

> This message was autogenerated

**Comment by advl** [comment]
Do we expect it to be available : 
1. In fullscreen mode ? 
2. In mdx stories ?
3. In docs ?

(cc @lyubomir-popov )

**Comment by bartaz** [comment]
Another consideration is having a keyboard shortcut (to be able to toggle this when button is not available).
